stdlib/InteractiveUtils/docs/src/index.md

```diff
@@ -1,5 +1,7 @@
 # Interactive Utilities
 
+This module is intended for interactive work. It is loaded automaticaly in [interactive mode](@ref command-line-options).
```

Comment:
```suggestion
The functionality documented on this page is defined in the `InteractiveUtils` standard library module. This module is intended for interactive work and it is loaded automatically in [interactive mode](@ref command-line-options), for example when working in the REPL.
```

Reply:
> The functionality documented on this page is defined in the `InteractiveUtils` standard library module.

I would have thought that is apparent from the context, like for all other modules.